docs/spec/delivery.md

```diff
@@ -16,7 +16,7 @@ subscription configuration member `DeadLetterPolicy`.
 The Knative dead letter policy is specified through the following parameters in
 the Knative Eventing delivery spec:
 
-- `DeadLetterSink`: We only allow special URLs as the dead letter sink, of the
+- `DeadLetterSink`: We only allow specific URLs as the dead letter sink, of the
```

Comment:
```suggestion
- `DeadLetterSink`: We only allow URLs of the form `pubsub://[dead_letter_sink_topic]` as the dead letter sink. 
```

What do you think if we change this sentence to this?

Reply:
Sounds good, done.